core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -6976,7 +6976,7 @@ GROUP by deptno, job]]>
         </Resource>
         <Resource name="plan">
             <![CDATA[
-LogicalProject(JOB_NAME=[CASE(OR(=($1, '810000'), =($1, '820000')), $1, 'error':VARCHAR(10))], EXPR$1=[$2])
+LogicalProject(JOB_NAME=[CASE(SEARCH($1, Sarg[_ISO-8859-1'810000', _ISO-8859-1'820000']:CHAR(6)), $1, 'error':VARCHAR(10))], EXPR$1=[$2])
```

Comment:
Could we trim the default encoding when dumping string literals within `Sarg`?

Reply:
Good idea. I'll do that.